client/apps/city_issues/models/users.py

```diff
@@ -73,16 +79,20 @@ def password(self):
         return self.hashed_password
 
     @password.setter
-    def set_hashed_password(self, raw_password):
-        super(User, self).set_password(raw_password)
+    def password(self, raw_password):
+        self.set_password(raw_password)

```

Comment:
why we use this function ? its only call "set_password()" ?
why we cant use just "set_password()"

Reply:
impossible, property requires a setter and a setter with the same name, if i got ur point

Reply:
https://docs.djangoproject.com/en/1.11/topics/auth/customizing/#specifying-a-custom-user-model
https://docs.python.org/2/howto/descriptor.html#properties